sdk/digitaltwins/Azure.DigitalTwins.Core/tests/QueryTests.cs

```diff
@@ -48,7 +48,15 @@ public async Task Query_ValidQuery_Success()
 
                 // Construct a query string to find the twins with the EXACT model id and provided version. If EXACT is not specified, the query
                 // call will get all twins with the same model id but that implement any version higher than the provided version
-                string queryString = $"SELECT * FROM digitaltwins WHERE IS_OF_MODEL('{roomModelId}', EXACT) AND IsOccupied = true";
+                string queryString = new DigitalTwinsQueryBuilder()
```

Comment:
Just capture the QueryBuilder Object and pass it on to the API as the QueryBuilder object.